sql/ddl_api.sql

```diff
@@ -213,6 +213,7 @@ CREATE OR REPLACE PROCEDURE @extschema@.refresh_continuous_aggregate(
     continuous_aggregate     REGCLASS,
     window_start             "any",
     window_end               "any",
-    force                    BOOLEAN = FALSE
+    force                    BOOLEAN = FALSE,
+    options                  JSONB = NULL
```

Comment:
What if the `force` should be included into `options` json field and just create an wrapper to build the json with this option?

Reply:
As discussed, I'll allow "force" option in `options` (should probably have a different name to be more explicit about what is forced, e.g., `force_refresh`) and use the `force` option to set this parameter. This will allow us to deprecate the `force` option and remove it in the future.

Reply:
This turned out to be a larger job since we are lacking functions to work with the options format. I will take this as a separate PR.